files/filter.d/wordpress.conf

```diff
@@ -0,0 +1,23 @@
+# Wordpress filter
```

Comment:
```suggestion
# Wordpress filter (this file is managed by Ansible - do not edit!)
```
It would be useful to be able to configure filters using Ansible as well. If they are in the role, I need to change the role each time, or worse, fork it if I don't have push access.